scripts/utils/default_yargs.js

```diff
@@ -0,0 +1,12 @@
+module.exports = require("yargs")
+  .version(false)
+  .strict()
+  .help("help")
+  .epilog(
+    "Make sure that you have an RPC connection to the network in consideration. For network configurations, please see truffle-config.js"
+  )
+  .option("network", {
+    type: "string",
+    describe: "network where the script is executed",
+    choices: ["rinkeby", "mainnet"],
```

Comment:
While we usually don't test against development, we could still offer it here

Reply:
The issue is that all the scripts rely on the multisig web interface, they will surely not work on development, so I didn't want to offer it explicitly. (It would be great to have a local version of the multisig, however!)
We can still test the scripts on development however: the `network` argument is not required, so by omitting it it automatically runs on development. This is also how I tested this PR.

Reply:
I am also running into this issue with the synthetix integration. Their npm package only exposes deployed versions of their system from which we can query their onchain price feed and token addresses (etc..). However, this will be untestable on the development network without depending on their entire contracts repo and deploying their entire system on every test.

Of course, this comment is not entirely relevant to this PR, I just needed to mentioned it somewhere and have not yet come up with a scheme for testing their stuff locally.